File: UnitTestsBD/UnitTestsBD/Models/Goals.cs
using Supabase.Postgrest.Attributes;
using Supabase.Postgrest.Models;

namespace UnitTestsBD.Models
{
    [Table("Goals")]
    public class Goals : BaseModel
    {
        [PrimaryKey("id")]
        public Guid Id { get; set; }

        [Column("name")]
        public string Name { get; set; }

        [Column("description")]
        public string Description { get; set; }

        [Column("id_sphere")]
        public int IdSphere { get; set; }

        [Column("status")]
        public bool Status { get; set; }

        [Column("id_user")]
        public Guid IdUser { get; set; }
    }

}

File: MakeYourselfApp/app/src/main/java/com/example/makeyourselfapp/models/database/Goals.kt
package com.example.makeyourselfapp.models.database

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Goals(
    var id: String = "",
    var name: String? = "",
    var description: String? = "",
    @SerialName("id_sphere")
    var idSphere: Int? = null,
    var status: Boolean = false,
    @SerialName("id_user")
    var idUser: String = ""
)